### ISSUE TITLE
testing interactive event: add focus to buttons

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#3145]
**URL:** moderator view
**user:** moderator
**expected behaviour:** I can use it without a mouse
**behaviour:** there is no focus for the three filter-icons and the focus for the category-filter is hardly visible
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?
<img width="291" alt="Bildschirmfoto 2020-09-22 um 17 11 00" src="https://user-images.githubusercontent.com/35491681/93907257-659dfe80-fcfd-11ea-9543-8899d940224c.png">
